docs/en/sql-reference/statements/select/order-by.md

```diff
@@ -221,3 +221,70 @@ returns
 │ 1970-03-12 │ 1970-01-08 │ original │
 └────────────┴────────────┴──────────┘
 ```
+
+## OFFSET FETCH Clause {#offset-fetch}
+
+`OFFSET` and `FETCH` allow you to retrieve just a portion of the rows that are generated by the rest of the query.
+
+``` sql
+OFFSET offset_row_count {ROW | ROWS} FETCH {FIRST | NEXT} fetch_row_count {ROW | ROWS} {ONLY | WITH TIES}
```

Comment:
Does FETCH the mandatory part of a query? Can we use OFFSET separatedly? If FETCH is an optional part of a query you must show it in the syntax description by square brackets.